### PR TITLE
Fixes #17643 - Add check to verify capsule tar

### PR DESCRIPTION
**Pull request by chris1984**
Results of my testing:

https://paste.fedoraproject.org/505514/15784011/

### CODE REVIEW

File: hooks/pre_validations/12-check_capsule_tar.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def error(message)
+  say message
+  logger.error message
+  kafo.class.exit 101
+end
+
+if param('capsule', 'certs_tar') && (certs_tar = param('capsule', 'certs_tar').value)
+  unless File.file?(certs_tar)
+    error 'The capsule tar file generated by the server is not present in the path specified, exiting.'
```
Comment (stbenjam) [inline hooks/pre_validations/12-check_capsule_tar.rb]: I'd say certs tar here

Comment (stbenjam) [inline hooks/pre_validations/12-check_capsule_tar.rb]: I'd also actually give the path that was specified, gives the user a chance to see if they fat fingered it. Something similar to 

```ruby
 error "The certs tar was not found at #{certs_tar}, exiting."
````

**Comment by stbenjam** [comment]
Otherwise :+1:

**Comment by chris1984** [comment]
@stbenjam thanks for the review :) Fixed the requested changes.

**Comment by stbenjam** [comment]
Thanks @chris1984 !

**Pull request merged**